stable/kommander/templates/pre-deletion-purger.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: purger-config
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-2"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+data:
+  purge.sh: |
+    #!/bin/bash
+    for ft in $(kubectl get federatedtypeconfigs -n kommander -o jsonpath='{range .items[*]}{.spec.federatedType.kind}.{.spec.federatedType.group}{"\n"}{end}') ; do
```

Comment:
What about the federated resources created in namespace kommander-system?

Reply:
Do we need to delete them?

Reply:
This line only searches for `FederatedTypeConfigs` and the Kommander chart does not create any `FederatedTypeConfigs` outside of the `kommander` namespace.

Reply:
True!